lib/use_case/month_calendar_get_events_use_case.dart

```diff
@@ -14,8 +16,19 @@ class MonthCalendarGetEventsUseCase {
     final startOfCurrentMonth = DateTime(date.year, date.month);
     final weekdayOfFirstDay = startOfCurrentMonth.weekday;
     final endOfCurrentMonth = DateTime(date.year, date.month + 1);
-    final daysBetween =
+
+    var daysBetween =
         startOfCurrentMonth.calculateDaysBetween(endOfCurrentMonth);
+
+    final isFirstDayOfNewMonthMonday = endOfCurrentMonth.weekday == 1;
+
+    if (!isFirstDayOfNewMonthMonday) {
+      final diff = DateTime.sunday - endOfCurrentMonth.weekday;
+      for (var i = 0; i <= diff; i++) {
+        daysBetween++;
+      }
+    }
```

Comment:
Ten kod uzupełnia ostatni row do końca tj. np. jeśli miesiąc się kończy 31 sierpnia w sobotę, to jeszcze dochodzi 1 dzień, 1 września. Myślę, że całkiem spoko opcja i zostawiamy nieopcjonalne, czyli tak jak dodałeś, @KrzysztofZelekk.